### DISCUSSION TITLE
[MU4 Issue] Use "measure" instead of "bar" in US English

### DISCUSSION THREAD

**Discussion by MarcSabatella** [musescore/MuseScore#11916] [General]
Mostly we've been consistent about the measure / bar distinction (US vs UK).  But a couple of places I've noticed in properties use "bar" even in US - "Bar width" in the Appearance section, and "Reset bar numbers" for section breaks.

BTW, "bar width" is, as far as I can tell, really just an alias for what is called "stretch" everywhere else?  I get that "stretch" is pretty mysterious, but "width" is inaccurate, since the default is 1 everywhere even for measures with very different widths.  I don't see introducing a less accurate alternative label in only one of the places this is exposed to be a way of reducing confusion.  Accurate and maybe somewhat more consistent with other terminology in MuseScore might be "measure width scaling", although that phase is not easy to love either.

**Comment by Tantacrul** [comment]
Moved this to a discussion. Any strings people want to change should be submitted prior to string freeze. Potentially, Bar/Measure disparities can be fixed after string freeze.

**Comment by MarcSabatella** [comment]
Indeed, changing bar to measure wouldn't affect translations a bit, so it can wait.

Are you saying this discussion is a place to track/propose other changes?  Not sure.

I see the change from "tempo ranged changed" to "gradual tempo change" is in already.  I know there are a bunch of possibly pending changes as a result of https://github.com/musescore/MuseScore/issues/8746

I don't know that there was ever a firm decision on the sub-beam tooltips, but they are still inconsistent between the Properties panel and the Palette - the first one is "Beam 16th sub" in the palettes, but the exact same icon is "Begin 32" in Properties.

The new feather icons seem to be missing tooltips.

**Reply by Jojo-Schmitz** [reply]
> Indeed, changing bar to measure wouldn't affect translations a bit, so it can wait.

Huh? Of course it would, all strings containg "bar" would need a re-translation once changed to "measure"

**Reply by MarcSabatella** [reply]
Hmm, maybe I don't understand the translation tools so well.  Are you saying that even fixing a typo in a string deletes all existing translations so they have to be re-entered from scratch, with no way to override that?  I'd have assumed the tools would be smart enough to allow one to choose to keep the existing translation if desired.  And surely the existing translation *would* still be good regardless of which English synonym was used.

**Reply by Jojo-Schmitz** [reply]
Yes, mostly. Sometimes it suggests the old translation

**Reply by Jojo-Schmitz** [reply]
Actually how could it? Assume the string "white", which needs to get changed to "black", of course it now needs to get retranslated. Or if you want a real  simple typo, think "back" vs. "black".
Same for "bar" and "measure".
There's no Thesaurus build into Transifex ;-)

**Reply by Jojo-Schmitz** [reply]
In this case though we'd have the en_US translation to the rescue (and that is the main point for having it in the first place).
Still we'd better get the strings right from the start and surely before string freeze.

**Comment by Jojo-Schmitz** [comment]
Seems "Reset measure numbers for new section" is the only one left, now also mentioned in issue #13164 and fixed in PR #12861
